The PR build failed. Its weird error about VM
```
[ERROR] Error occurred in starting fork, check output in log
[ERROR] Process Exit Code: 1
[ERROR] org.apache.maven.surefire.booter.SurefireBooterForkException: The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
```

--- a/common/src/test/resources/common-test-properties/common.properties
+++ b/common/src/test/resources/common-test-properties/common.properties
@@ -18,3 +18,5 @@
 # This is used only for the PropertySourceOverrideTest
 test.property.source=commonvalue
 shared.override.test=commonvalue
+
+auto.correct.sequence.generator.inconsistencies=true

--- a/integration/src/test/resources/config/bc/overrideprops/common.properties
+++ b/integration/src/test/resources/config/bc/overrideprops/common.properties
@@ -17,3 +17,5 @@
 
 # Test file to be overridden by SystemPropertiesTest
 property.file.override.test=propertyfile
+
+auto.correct.sequence.generator.inconsistencies=true

